.docker/intelmq-full-dev/update.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+echo "Installing requirements for bots in dev repository"
+for file in $(find /opt/dev/mybots -name "*REQUIREMENTS.txt"); do pip3 install -r $file; done
+
+# Backup Original BOTS
+cp /opt/intelmq/intelmq/bots/BOTS /opt/intelmq/intelmq/bots/BOTS.bk
+
+echo "Merge your BOTS file with BOTS"
+python3 /opt/dev/merge_BOTS.py "/opt/dev/mybots/BOTS" "/opt/intelmq/intelmq/bots/BOTS" "/opt/intelmq/intelmq/bots/BOTS"
```

Comment:
Just as an information: The BOTS file is obsolete in IntelMQ 3.0.0. So in three months, you'll need to update this logic here (at the latest, otherwise the building will fail).

Reply:
Great news! I keep it in mind! Thank you

Reply:
Now i check if BOTS exists